frontend/src/apis/segments.ts

```diff
@@ -48,10 +52,45 @@ export const updateSegment = async (segmentObj: Segment) => {
 export const deleteSegment = async (params: {
   segmentId: string;
   appId: string;
-  projectId: string;
 }) => {
   return (await apiRequest(
     'delete',
-    `/segments/${params.segmentId}?appId=${params.appId}&projectId=${params.projectId}`
+    `/segments/${params.segmentId}?appId=${params.appId}`
+  )) as ApiResponse<{ segmentId: string }>;
+};
+
+export const getSegmentJobs = async (params: { segmentId: string }) => {
+  return (await apiRequest(
+    'get',
+    `/segments/${params.segmentId}/jobs`
+  )) as ApiResponse<SegmentJobStatusItem[]>;
+};
+
+export const getExportFileS3Url = async (params: {
+  projectId: string;
+  appId: string;
+  segmentId: string;
+  jobRunId: string;
+}) => {
+  return (await apiRequest(
+    'get',
+    `/segments/${params.segmentId}/jobs/${params.jobRunId}/exportS3Url?projectId=${params.projectId}&appId=${params.appId}`
```

Comment:
Looks like we have different API styles,

- /analytics/:projectId/app/:appId/segments/:segmentId/details
- /segments/${params.segmentId}?appId=${params.appId}
- /segments/${params.segmentId}/jobs/${params.jobRunId}/exportS3Url?projectId=${params.projectId}&appId=${params.appId}

We might consolidate them in another PR.

Reply:
`/analytics/:projectId/app/:appId/segments/:segmentId/details` is the URL, not the API.  

Our API routers are following the pattern, from which we can tell the APIs classification.
```
// routers
app.use('/api/env', router_env);
app.use('/api/project', router_project);
app.use('/api/app', router_app);
app.use('/api/pipeline', router_pipeline);
app.use('/api/plugin', router_plugin);
app.use('/api/metadata', router_metadata);
app.use('/api/reporting', router_reporting);
app.use('/api/user', router_user);
app.use('/api/segments', router_segment);
app.use('/api/system', router_system);
app.use('/api/traffic', router_traffic);
```
When I designed the segments APIs, I put the `jobRunId` which belongs to specific segment in the URI path, while adding `projectId` and `appId` as query params.